recipes/nd2-dask/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "nd2-dask" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nd2_dask-{{ version }}.tar.gz
+  sha256: a4dc0b129e24c855b7d51047d8c9d08bc8e1b36769a488fafd1ffe405d71507a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<3.10
```

Comment:
We don't usually set upper bounds. Why do we need this for 3.10? If a specific version doesn't work it's better to deliberately skip it with `>=3.6,!=3.10`, for example. Same in `run`.

Reply:
I think there are some issue with a deprecated import on a dependency

`from collections import Iterable` is removed in 3.10, there are a few recipes that have failed because of this. The issue with skip, is that this problem will continue in all subsequent versions unless the dependency is fixed (which should happend eventually)

Reply:
Got it, thanks!